nav2_system_tests/src/recoveries/spin/spin_recovery_tester.cpp

```diff
@@ -67,12 +81,18 @@ void SpinRecoveryTester::activate()
     throw std::runtime_error("Trying to activate while already active");
     return;
   }
-
-  while (!initial_pose_received_) {
-    RCLCPP_WARN(node_->get_logger(), "Initial pose not received");
-    sendInitialPose();
-    std::this_thread::sleep_for(100ms);
-    rclcpp::spin_some(node_);
+  if(!make_fake_costmap_){
+    while (!initial_pose_received_) {
+      RCLCPP_WARN(node_->get_logger(), "Initial pose not received");
+      sendInitialPose();
+      std::this_thread::sleep_for(100ms);
+      rclcpp::spin_some(node_);
+    }
+  }
+  else {
+      sendFakeFootprint();
```

Comment:
linting

Reply:
I'm going to stop marking linting errors, but please fix them all with ament_cpplint and ament_uncrustify